build.gradle

```diff
@@ -15,16 +15,13 @@ plugins {
 }
 
 dependencies {
-
-    implementation 'io.rest-assured:rest-assured:4.0.0'
-    compile group: 'org.json', name: 'json', version: '20180813'
-    
-    testCompile group: 'junit', name: 'junit', version: '4.12'
-    testCompile group: 'info.cukes', name: 'cucumber-java', version: '1.2.5'
+    implementation 'io.rest-assured:rest-assured:4.0.0'    
+    testCompile 'io.cucumber:cucumber-java:4.3.1'
     testCompile group: 'info.cukes', name: 'cucumber-picocontainer', version: '1.2.5'
-    testCompile group: 'info.cukes', name: 'cucumber-testng', version: '1.2.5'
+    compile group: 'io.cucumber', name: 'cucumber-testng', version: '4.7.1'
```

Comment:
I understand that Cxristian has dropped the use of io.cucumber in favor of info.cukes:cucumber-testng, so he can implement the html reports.

Reply:
The dependencies have been changed in order to the code to be able to run. With the previous dependencies, hooks seemed not to work or there was some incompatibility.

Reply:
Ok, I see. By the way, are you able to execute the runner via command line and generate the reports?

Anyway, I guess Cxristian should give us some insight about these dependencies.